util/mkmake.c

```diff
@@ -80,6 +80,10 @@ void process_makefile (const char *in_fname, const char *out_fname)
     }
   }
 
+  p = getenv ("CFLAGS");
+  if (!p) p = "";
+  fprintf (out, "USER_CFLAGS := %s\n", p);
```

Comment:
This look very GNU-make specific; `A := B`. Does e.g. nmake or wmake support this syntax?

Reply:
Didn't know that was gnu-specific.  You're right, others don't support this.  Will fix.